docs/docs-beta/docs/dagster-plus/getting-started.md

```diff
@@ -3,4 +3,63 @@ title: "Getting started with Dagster+"
 displayed_sidebar: "dagsterPlus"
 ---
 
-# Get started with Dagster+
\ No newline at end of file
+# Get started with Dagster+
```

Comment:
> This seems to fit more in line with What's Dagster+?

I originally wrote this all under the `architecture.md` heading but switched it right before pushing. My sense is that our current "hybrid architecture" docs are simultaneously too in the weeds (it's not really all that important that it's "grpc" or "graphql") and don't do a good job of explaining the different moving parts of Dagster+ (I think it takes a few reads to understand what the agent actually does). And I wanted a good place to contextualize what the agent actually does that I can link to from other agent guides.

I can move it back under the architecture heading for now. Or ditch entirely if we prefer the old doc.